### ISSUE TITLE
Elasticity too strong

### ISSUE THREAD

**Issue by eonist** [IsaacXen/IXRefreshableScrollView#6]
It takes too much effort to pull to refresh.

Is it possible to adjust the elasticity to be more loose and easier to pull to refresh. Sort of like iOS?

Thanks

**Comment by eonist** [comment]
Also, do you know any similar projects?

**Comment by IsaacXen** [comment]
Yes, I think it's possible to adjust the elasticity, but it requires you to override `scrollWheel(with:)` method and handle elastic scrolling completely.

It's not an easy task though, but I'll see what I can do, maybe as a new feature in future build.

---

There are some similar projects:

- [alexzielenski/ScrollToRefresh](https://github.com/alexzielenski/ScrollToRefresh)
- [adib/RefreshableScrollView](https://github.com/adib/RefreshableScrollView)

Also, it's worth checking UIKit implementations. Scroll view API is quite similar on both platform and you can get it working on macOS with little modification.

**Comment by eonist** [comment]
@IsaacXen Thanks for your insight! 

I think overriding the scroll wheel and doing the scrolling calculations your self  could work. I'll give it a go. 👍
